src/components/RoomView/RoomView.jsx

```diff
@@ -314,13 +311,7 @@ const RoomView = () => {
                 <p key={player.playerId}>{player.username}</p>
               ))}
             </Item>
-            {gameStatus !== 'ready' && (
-              <Item style={styles.sx.PlayerContainer}>
-                Turn:
-                {gameStatus}
-                <GuessesRemaining />
```

Comment:
we'll probably need to work together on the gameStatus component (it now contains <GuessesRemaining/>) since it was originally going to live here. open to your suggestions since you've been working on style! :) might be a minor merge conflict